mthamayor/auto-mart
# 166933192 Implement API consumption on car view

#### What does this PR do?
Users can Interact with the [car](https://mthamayor.github.io/auto-mart/car.html) page

#### Description of Task to be completed?
- The page should consume the following endpoints based on user interractions
  1. GET /car/{car_id}/
  2. DELETE /car/{car_id}/
  3. GET /car/{car_id}/flags
  4. GET /car/{car_id}/orders
  5. PATCH /car/{car_id}/status
  6. GET /car/{car_id}/
  7. PATCH ​/order​/{order_id}​/reject
  8. PATCH ​/order​/{order_id}​/accept

#### How should this be manually tested?
- Search for cars  
  For users: [here](https://mthamayor.github.io/auto-mart/search-cars.html)
  For admins:  [here](https://mthamayor.github.io/auto-mart/admin-search-cars.html)
- Click on a car and it should direct you to the car page.
- Depending on the user type (Admin, Seller, User), Controls button would be provided for consuming API endpoints

#### Any background context you want to provide?
None

#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/166933192

#### Screenshots (if appropriate)
![Screenshot (134)](https://user-images.githubusercontent.com/16121915/60433956-f3279700-9bfd-11e9-9dc5-2027e460c514.png)
![Screenshot (135)](https://user-images.githubusercontent.com/16121915/60433957-f3c02d80-9bfd-11e9-8df0-e20513bd42c5.png)
![Screenshot (136)](https://user-images.githubusercontent.com/16121915/60433959-f3c02d80-9bfd-11e9-960f-85b3ebf2efab.png)
![Screenshot (137)](https://user-images.githubusercontent.com/16121915/60433961-f458c400-9bfd-11e9-8680-92927e008061.png)
![Screenshot (138)](https://user-images.githubusercontent.com/16121915/60433962-f458c400-9bfd-11e9-9902-9705daf90eef.png)
![Screenshot (139)](https://user-images.githubusercontent.com/16121915/60433963-f4f15a80-9bfd-11e9-81ae-fafd72b35b63.png)
![Screenshot (140)](https://user-images.githubusercontent.com/16121915/60433965-f589f100-9bfd-11e9-96ce-1324a05d1c10.png)

#### Questions:
None
